docs/workflows/transitions/preprocess/preprocess/utils.py

```diff
@@ -211,3 +217,27 @@ def patch_empty_predictions(predictions, labels, no_empty_prediction_fields):
     return patched_predictions
 
 
+def filter_away_low_confidence_lines(predictions, field_config):
+    column_names = {
+        label: {column_name for column_name in config['fields']}
+        for label, config in field_config.items() if config.get('type') == 'lines'
+    }
+
+    for prediction in predictions:
+        label = prediction['label']
+        if field_config.get(label, {}).get('type') == 'lines':
+            line_predictions = []
+            for line in prediction['value']:
+                top_1_line = filter_by_top1(line, column_names[label])
```

Comment:
If I understand correctly how this works, I think this is a clearer name.
```suggestion
                top_1_predictions = filter_by_top1(line, column_names[label])
```